src/pages/my-profile/screens/MyProfileAdDetails/MyProfileAdDetailsTextArea/MyProfileAdDetailsTextArea.tsx

```diff
@@ -0,0 +1,64 @@
+import { Controller, useForm } from 'react-hook-form';
+import { useTranslations } from '@deriv-com/translations';
+import { TextArea, useDevice } from '@deriv-com/ui';
+
+type TMyProfileAdDetailsTextAreaProps = {
+    control: ReturnType<typeof useForm>['control'];
+};
+
+const MyProfileAdDetailsTextArea = ({ control }: TMyProfileAdDetailsTextAreaProps) => {
+    const { localize } = useTranslations();
+    const { isMobile } = useDevice();
+    const textSize = isMobile ? 'md' : 'sm';
+    const pattern = {
+        message: localize(
+            "Contact details can only include letters, numbers, spaces, and any of these symbols: -+.,'#@():;"
```

Comment:
shouldn't the fieldname be variable?

Reply:
fieldname? this is just for the error

Reply:
isn't the same validation used for instructions text area as well?

Reply:
yuppp thats why i put the whole pattern in a const variable, its the same for both

Reply:
no. i mean.. for the instructions field also won't it show the error "contact details can only include.... "

Reply:
i passed the pattern in the rules

Reply:
oh i just realised, ill fix this then